packages/map/src/map/index.jsx

```diff
@@ -13,6 +13,7 @@ const Map = ({
   className,
   children,
 
+  translations,
```

Comment:
I would go with `labels` (same as in react datepicker)

Reply:
i used tranlsations in cookiebanner

Reply:
i second `translations`, clearer